core/ir/x86/decode.c

```diff
@@ -1121,7 +1121,8 @@ read_instruction(byte *pc, byte *orig_pc, const instr_info_t **ret_info,
     } else if (info->type == EVEX_W_EXT) {
         /* discard old info, get new one */
         int code = (int)info->code;
-        int idx = (TEST(PREFIX_REX_W, di->prefixes) ? 1 : 0);
+        int idx = (TEST(PREFIX_REX_W, di->prefixes) ? 2 : 0) +
+            (TEST(PREFIX_EVEX_b, di->prefixes) ? 1 : 0);
```

Comment:
Yes, I would rename EVEX_W_EXT and evex_W_extensions.

The alternative is to split off a separate evex_b_extensions with another table lookup.  If the number of evex_W_extensions entries that care about evex.b is small, that may be nicer.  (If doing that, and if evex.b does make them invalid, we could do that via a REQUIRES_EVEX_B flag.)

Reply:
Ultimately 88 out of 265 entries don't vary at all on evex.b.

Reply:
OK, so the vast majority need all.  OK, seems reasonable how you have it.

Reply:
Is EVEX_WB_EXT and evex_WB_extensions fine? or do you have other preferred names?

Reply:
> Is EVEX_WB_EXT and evex_WB_extensions fine?

SGTM